hubmap/frontend/src/commons/human-svg-utils.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-plusplus */
+
+export function updateText(tspanId, txt) {
+  const spanEl = document.getElementById(tspanId);
+  while (spanEl.firstChild) {
+    spanEl.removeChild(spanEl.firstChild);
+  }
+  spanEl.appendChild(document.createTextNode(txt));
+}
+
+export function showToolTip(evt) {
+  const humanSvg = document.getElementById('human');
+  const tooltip = humanSvg.getElementById('tooltip');
+  const tooltipText = tooltip.getElementsByTagName('text')[0];
+  const tooltipRects = tooltip.getElementsByTagName('rect');
+  const CTM = humanSvg.getScreenCTM();
+  const mousex = (evt.clientX - CTM.e + 6) / CTM.a;
+  const mousey = (evt.clientY - CTM.f + 20) / CTM.d;
+  tooltip.setAttributeNS(null, 'transform', `translate(${mousex} ${mousey})`);
+  tooltip.setAttributeNS(null, 'visibility', 'visible');
+  const lines = evt.target.getAttributeNS(null, 'data-tooltip-text').split(',');
+  // eslint-disable-next-line prefer-destructuring
+  tooltipText.firstChild.data = lines[0];
+  updateText('tspanid', lines[1]);
+  const length = tooltipText.getComputedTextLength();
+  for (let i = 0; i < tooltipRects.length; i++) {
```

Comment:
maybe
```
tooltipRects.forEach((rect) => {
...
});
```

Reply:
it turned out at any given point of time: i.e. when any given mouse pointer move event, tooltipRectangle is always one. because for getElementsByTagName/Classname we always get HTML collection, we have a collection. Using Array.from(HTMLCollection object) was good, but did not require this loop, since, as per lot of tests, there is only 1 element in the Collection all the time. So just get [0] and no need to even loop. i.e. to say, there is always only single tooltip present at any given mouse pointer on screen.